en/django_forms/README.md

```diff
@@ -44,32 +44,58 @@ So once again we will create a link to the page, a URL, a view and a template.
 
 ## Link to a page with the form
 
-It's time to open `blog/templates/blog/base.html` in the code editor. In the `div` named `page-header`, we will add a link:
+It's time to open `blog/templates/blog/base.html` in the code editor. In the `div` named `container` inside `page-header` `div` tag, we will add a link:
 
 {% filename %}blog/templates/blog/base.html{% endfilename %}
 ```html
-<a href="{% url 'post_new' %}" class="top-menu"><span class="glyphicon glyphicon-plus"></span></a>
+<a href="{% url 'post_new' %}" class="top-menu">
+    <svg width="1em" height="1em" viewBox="0 0 16 16" class="bi bi-file-earmark-plus" fill="currentColor" xmlns="http://www.w3.org/2000/svg"><path d="M4 0h5.5v1H4a1 1 0 0 0-1 1v12a1 1 0 0 0 1 1h8a1 1 0 0 0 1-1V4.5h1V14a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V2a2 2 0 0 1 2-2z"/><path d="M9.5 3V0L14 4.5h-3A1.5 1.5 0 0 1 9.5 3z"/><path fill-rule="evenodd" d="M8 6.5a.5.5 0 0 1 .5.5v1.5H10a.5.5 0 0 1 0 1H8.5V11a.5.5 0 0 1-1 0V9.5H6a.5.5 0 0 1 0-1h1.5V7a.5.5 0 0 1 .5-.5z"/></svg>
+</a>
 ```
 
-Note that we want to call our new view `post_new`. The class `"glyphicon glyphicon-plus"` is provided by the bootstrap theme we are using, and will display a plus sign for us.
+Note that we want to call our new view `post_new`. The [svg icon](https://icons.getbootstrap.com/icons/file-earmark-plus/) is provided by the [Bootstrap Icons](https://icons.getbootstrap.com/) and it will display a page icon with plus sign.
 
-After adding the line, your HTML file should now look like this:
+The addition of SVG code for the icon made it look very complicated. So we can try a different approach to achieve the same result.
+
+Let's save the svg code in a separate file. Save the following code in the file `blog/templates/blog/icons/add-post.svg`
```

Comment:
I'd rather not make the tutorial longer than it already is. I'd also rather not introduce yet another language (We already have at least 4: command line (Bash or CMD), Python, HTML and CSS.) and thus would prefer to treat SVG just as an image format, without showing the SVG file's source code.

So can we let participants
- use the SVG as file without first having it inline in the template's HTML?
- download the whole SVG file (e.g. from https://raw.githubusercontent.com/twbs/icons/main/icons/file-earmark-plus.svg) and save it at the right location, rather than copying or typing its textual content?

For interested participants, we can link to https://en.wikipedia.org/wiki/Scalable_Vector_Graphics or some other information about the format. That'd be less overwhelming than the wall of ~~text~~ code we'd place before them with the changes currently suggested here.

Reply:
Will do that

Reply:
Updated https://github.com/DjangoGirls/tutorial/pull/1681/commits/56d846c0746237e1cf3f2758d5cf3d5d67cf4bce

Reply:
Please check the updated changes in the following sections:

https://django.nikz.in/en/django_forms/#link-to-a-page-with-the-form

https://django.nikz.in/en/django_forms/#edit-form